okhttp-tests/src/test/java/okhttp3/RequestTest.java

```diff
@@ -175,7 +175,7 @@
   }
 
   @Test public void headerForbidsControlCharacters() throws Exception {
-    assertForbiddenHeader(null);
+    assertNullHeader(null);
```

Comment:
You can just delete this line and convert the new method into a test named `headerForbidsNullArguments`